ActindoWorkFlow/Operation/03_CompareWorkflows.md

```diff
@@ -0,0 +1,178 @@
+# Compare the workflows versions
+
+The comparison function of the workflow enables to compare different workflow versions. By doing this, you can compare the JSON code lines of two different workflow versions, whereby all differences between the relevant code lines are highlighted.  
+
+You can use this function to do the following: 
+- Compare a specific workflow version to any other version of that workflow available on the current instance.
+- Compare a specific workflow version to any workflow version available on another instance. 
+- Change, insert, or delete code lines. This is possible, because you can edit the JSON code directly.  
+- Display change logs. By doing it, you are able to do software-audits.
+
+## Open the comparison window
+
+Open the comparison window to compare different workflow versions. There are two different ways to open it.
+
+#### Prerequisites
+
+You have the required rights to edit a workflow.
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW*
+
+![Workflows](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+Select the workflow for which you want to do a comparison.   
+The *Workflow versions* view is displayed.
+    
+![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowVersions.png "[Workflow versions]")
+
+**Open the comparison window via the Workflow versions view**
+
+1. Click the checkbox at the desired version you want to use for comparison.   
+    The editing toolbar is displayed.
+    
+2. Click the [COMPARE] button.  
+    The comparison window is displayed. 
+
+    ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowComparison.png "[Workflow]")  
+
+**Open the comparison window in the workflow editor**   
+
+1. Select the workflow version.   
```

Comment:
Vorschlag: 1. Select a workflow version. / 1. Select the workflow version you want to use for compare / you want to compare (to).

"a" in dem Sinne von "any". Also, man öffnet eine beliebige Version und von kann man eine zweite Version öffnen, mit der man diese erst geöffnete Version vergleicht, oder?